src/textual/_layout.py

```diff
@@ -25,6 +27,7 @@ class WidgetPlacement(NamedTuple):
     region: Region
     widget: Widget | None = None  # A widget of None means empty space
     order: int = 0
+    fixed: bool = False
```

Comment:
Sort of equivalent to `position: fixed` in CSS. Basically excludes a widget from scrolling.